config/fcos/v1_6_exp/validate_test.go

```diff
@@ -237,6 +237,141 @@ func TestValidateGrubUser(t *testing.T) {
 	}
 }
 
+func TestValidateMountPoints(t *testing.T) {
+	tests := []struct {
+		in      Config
+		out     error
+		errPath path.ContextPath
+	}{
+		// valid config (has prefix "/etc" or "/var")
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path: util.StrToPtr("/etc/foo"),
+							},
+							{
+								Path: util.StrToPtr("/var"),
+							},
+						},
+					},
+				},
+			},
+		},
+		// invalid config (path name is '/')
```

Comment:
We're not setting `out`, so this is actually testing that the config is valid.  Needs `WithMountUnit`.

Reply:
Not done.

Reply:
My bad i've commited this will push when all changes are made:
```
// invalid config (path name is '/')
		{
			in: Config{
				Config: base.Config{
					Storage: base.Storage{
						Filesystems: []base.Filesystem{
							{
								Path:          util.StrToPtr("/"),
								WithMountUnit: util.BoolToPtr(true),
							},
						},
					},
				},
			},
			out:     common.ErrMountPointForbidden,
			errPath: path.New("yaml", "storage", "filesystems", 0, "path"),
		},
```
Ran tests and it passes also